test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1

```diff
@@ -165,4 +165,54 @@ Describe "DSC MOF Compilation" -tags "CI" {
         Remove-Item -Force -Recurse -Path WordPressServer
     }
 
+	It "Should be able to compile a MOF from a basic configuration on Windows" -Skip:($IsMacOS -or $IsLinux) {
+        [Scriptblock]::Create(@"
+        configuration DSCTestConfig
+        {
+            Import-DscResource -ModuleName PSDesiredStateConfiguration
+            Node "localhost" {
+                File f1
+                {
+					        DestinationPath = "$env:SystemDrive\\Test.txt";
+					        Ensure = "Present"
+                }
+            }
+        }
+
+        DSCTestConfig
+"@).Invoke()
+
+        Remove-Item -Force -Recurse -Path DSCTestConfig
+    }
+
+	It "Should be able to compile a MOF from a configuration with multiple resources on Windows" -Skip:($IsMacOS -or $IsLinux) {
+        [Scriptblock]::Create(@"
+        configuration DSCTestConfig
+        {
+            Import-DscResource -ModuleName PSDesiredStateConfiguration
+            Node "localhost" {
+                File f1
+                {
+                  DestinationPath = "$env:SystemDrive\\Test.txt";
+					        Ensure = "Present"
+                }
+				        Script s1
+				        {
+					        GetScript = {return @{}}
+					        SetScript = "Write-Verbose Hello"
+					        TestScript = {return $false}
+				        }
+				        Log l1
+				        {
+					        Message = "This is a log message"
+				        }
+            }
+        }
+
+        DSCTestConfig
+"@).Invoke()
```

Comment:
It might be good to put this in the form of `{[Scr...Invoke()}|should not throw` and/or verify the expected file gets created.

Reply:
I've submitted a PR to your branch to make this change:  https://github.com/Indhukrishna/PowerShell/pull/1
I'll make the change directly, once the tests pass if you would like me to.

Reply:
Tests have passed on Linux and Mac.  I'm going to push to your branch.